### ISSUE TITLE
Reserved field "null" not correctly parsed with the graphql-maven-plugin

### ISSUE THREAD

**Issue by hohler** [graphql-java-generator/graphql-maven-plugin-project#188]
Versions tested: 1.18, 2.0

A graphQL field named `null` is not correctly transformed.
I've seen that it works correctly e.g. for fields named `package`

This is an example from the linear.app GraphQL schema.

generated class:

```
/**
 * Null constraint. Matches any non-null values if the given value is false, otherwise it matches null values.
 */
@JsonProperty("null")
@GraphQLScalar( fieldName = "null", graphQLTypeSimpleName = "Boolean", javaClass = java.lang.Boolean.class, listDepth = 0)
java.lang.Boolean null;
```

graphql Schema:

```
input NullableDateComparator {
  """
  Null constraint. Matches any non-null values if the given value is false, otherwise it matches null values.
  """
  null: Boolean
}
```

**Comment by etienne-sf** [comment]
Hello,

Yes, you're right. 

The ´null´ keyword is missing in the keywords list
:( 

This will be solved in the next version. 

Étienne

**Comment by etienne-sf** [comment]
Solved in the 2.0 release
